external/book/content/book/en/v2/Git-Internals-The-Refspec.html

```diff
@@ -38,8 +38,8 @@ <h2 id="_refspec">The Refspec</h2>
 </div>
 </div>
 <div class="paragraph">
-<p>The format of the refspec is, first, an optional <code>`, followed by `&lt;src&gt;:&lt;dst&gt;`, where `&lt;src&gt;` is the pattern for references on the remote side and `&lt;dst&gt;` is where those references will be tracked locally.
-The `</code> tells Git to update the reference even if it isn’t a fast-forward.</p>
+<p>The format of the refspec is, first, an optional <code>`</code>, followed by <code>&lt;src&gt;:&lt;dst&gt;</code>, where <code>&lt;src&gt;</code> is the pattern for references on the remote side and <code>&lt;dst&gt;</code> is where those references will be tracked locally.
+The <code>`</code> tells Git to update the reference even if it isn’t a fast-forward.</p>
```

Comment:
As per https://github.com/git/git-scm.com/blob/0d41ffb2cd45fcb06f0f46e1f6d9a7f99bca6f26/external/book/content/book/en/v2/Git-Internals-The-Refspec.html#L2, please do not edit this file directly.

Seeing that the [source](https://github.com/progit/progit2/blob/652208f3f9ba3f31a846cf54b4b1aa682dd72e09/book/10-git-internals/sections/refspec.asc?plain=1#L21) is correct, we need to figure out why it is not rendered correctly instead.

The script that does the rendering code is located here: https://github.com/git/git-scm.com/blob/HEAD/script/update-book2.rb. I'll have a go at figuring out what is going on here, and circle back.

Reply:
@hanu9257 I figured it out: It is a regression in AsciiDoctor (I've reported it [here](https://github.com/asciidoctor/asciidoctor/issues/4624)). Previously, the ProGit book was rendered using AsciiDoctor v2.0.16, now it uses v2.0.23 (the latter having that bug).

How about resolving it like this?

```diff
diff --git a/script/update-book2.rb b/script/update-book2.rb
index 36baea4b5..a25aca5f1 100644
--- a/script/update-book2.rb
+++ b/script/update-book2.rb
@@ -74,6 +74,9 @@ def genbook(language_code, &get_content)
   content = expand(initial_content, "progit.asc") { |filename| get_content.call(filename) }
   # revert internal links decorations for ebooks
   content.gsub!(/<<.*?\#(.*?)>>/, "<<\\1>>")
+  # work around AsciiDoctor v2.0.19 and later mis-interpreting multiple `+` as
+  # indicating the start or the end of literal monospace.
+  content.gsub!(/`\+`/, "`{plus}`")
 
   asciidoc = Asciidoctor::Document.new(content, attributes: { "lang" => language_code })
   html = asciidoc.render
```

Reply:
Oh sorry for the mistake. I'll also find out what's happening.